modules/attachments/README.md

```diff
@@ -0,0 +1,83 @@
+# SCC Profile Attachment
+
+This module creates SCC Profile's (https://cloud.ibm.com/docs/security-compliance?topic=security-compliance-build-custom-profiles&interface=ui). A profile attachment is an association between the set of resources that you want to evaluate and a profile that contains the specific controls that you want to evaluate.
+
+The module supports the following actions:
+- Create SCC Profile Attachment
+
+### Usage
+
+```hcl
+provider "ibm" {
+  ibmcloud_api_key = "XXXXXXXXXX" # pragma: allowlist secret
+  region           = "us-south"
+}
+
+# - SCC Profile Attachment
+module "create_scc_profile_attachment " {
+  source           = "../../profile/."
```

Comment:
this should use the full source here like consumers would, not a local directory